### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by niquerio**
Primarily adding renew_all functionality to Alma Loans.

### CODE REVIEW

File: views/shelf.erb [hunk 1]
```diff
@@ -68,6 +71,9 @@ Renew all could be too slow. Maybe reconsider.
           <span>&#183; <%= loan.author %></span>
         </td>
         <td>
+          <% if loan.message? %>
+            <span class="action-<%=loan.message_status%>"><%= loan.message%></span>
+          <% end %>
```
Comment (niquerio) [inline views/shelf.erb]: The inline messages do not look good. Halp.

Comment (jonearley) [reply views/shelf.erb]: Sure thing! I'll get to this sometime this week, hopefully, tomorrow. Thanks, @niquerio!

Comment (erinesullivan) [reply views/shelf.erb]: I saw the already existing but unused `./css/_tags.scss` styles. I updated the `span` class and `tags--error` class name to `tags--fail`. I wasn't sure if it was better to change the class name or to change all mentions of `:fail` to `:error`.

**Review by erinesullivan — APPROVED** [review]
The tests pass. Code looks good. I tried to renew all and was unable to. It gave a message of how many were renewed and listed of all the items I could not renew.

**Pull request merged**